moved/src/types/session_id.rs

```diff
@@ -100,11 +100,6 @@ fn u8_chain_id(genesis_config: &GenesisConfig) -> u8 {
     if genesis_config.chain_id == CHAIN_ID {
         1
     } else {
-        // TODO: Should have some generic fallback algorithm here
-        // (e.g. just take the least significant byte) and a feature
-        // flag to enable it. This would allow people to set their own
-        // custom chain ids. This is not launch-critical since for now
-        // we are only picking chain ids internally.
-        panic!("Unknown chain id: {}", genesis_config.chain_id);
+        genesis_config.chain_id.to_le_bytes()[0]
```

Comment:
nit: I see this U256 → U64 conversion a lot. Any way we can make it a method like `to_u64_lossy()` cause `le` vs `be` could be mishandled?

Reply:
This is actually conversion to u8, so the occurrence here is unique in our code. The issue is that Aptos chose u8 to represent chain id for some reason.